ui/src/app/edge/live/energymonitor/chart/section/consumption.component.html

```diff
@@ -18,7 +18,7 @@
         <ng-container *ngIf="valueText; else no_text">{{valueText}}</ng-container>
         <ng-template #no_text>-</ng-template>
     </svg:text>
-    <svg:image [attr.x]="square.image.x" [attr.y]="square.image.y" [attr.width]="square.image.length"
-        [attr.height]="square.image.length" [attr.xlink:href]="square.image.image">
-    </svg:image>
+    <svg:use style="color: var(--ion-color-warning)" [attr.x]="square.image.x" [attr.y]="square.image.y" [attr.width]="square.image.length"
+        [attr.height]="square.image.length" [attr.xlink:href]="square.image.image + '#root'">
```

Comment:
please change to this: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href